src/hooks/bridge/type.ts

```diff
@@ -0,0 +1,15 @@
+export type LISTENLING_WEBVIEW_MESSAGE_KEY = 'foo' | 'bar';
+
+export type POST_WEBVIEW_MESSAGE_KEY = 'baz';
+
+export interface ListeningAppMessageData {
+  type: LISTENLING_WEBVIEW_MESSAGE_KEY;
+  data: unknown;
+  [key: string]: unknown;
+}
```

Comment:
요거 제너럴로 만들어주면 좋을 것 같기도 하네요

```suggestion
export interface ListeningAppMessageData<D = any> {
  type: LISTENLING_WEBVIEW_MESSAGE_KEY;
  data: D;
}
```

Reply:
제너릭으로 `data`의 타입을 지정할 수 있는 것이죠 ??! 좋은 거 같아요!!

일반적인 데이터를 `data`에, 추가적인 요소를 받을 수 있게 `[key: string]` 이렇게 작성해 보았는데 이것도 더 type safe하게 쓸 수 있을 지는 잘 모르겠네요 ㅠㅠ!

요 부분 기억해 두었다가, 실제 사용하는 과정에서 어떤 경우가 많은 지 확인 후 수정해 보면 좋을 거 같아서, 일단 미래의 저에게 일을 미뤄볼게요!